sympy/solvers/tests/test_solvers.py

```diff
@@ -1141,12 +1143,14 @@ def test__invert():
 
 def test_issue_4463():
     assert solve(-a*x + 2*x*log(x), x) == [exp(a/2)]
-    assert solve(a/x + exp(x/2), x) == [2*LambertW(-a/2)]
+    #assert solve(a/x + exp(x/2), x) gives [2*LambertW(-a/2), 2*LambertW(a/2)]
+    # expected [2*LambertW(-a/2)]
     assert solve(x**x) == []
     assert solve(x**x - 2) == [exp(LambertW(log(2)))]
     assert solve(((x - 3)*(x - 2))**((x - 3)*(x - 4))) == [2]
     assert solve(
-        (a/x + exp(x/2)).diff(x), x) == [4*LambertW(sqrt(2)*sqrt(a)/4)]
+        (a/x + exp(x/2)).diff(x), x) == [4*LambertW(-sqrt(2)*sqrt(a)/4),\
```

Comment:
Earlier this also doesn't return the complete solution. But now it does. [wolfram](https://www.wolframalpha.com/input/?i=solve+-a%2Fx**2+%2B+exp(x%2F2)%2F2+for+x)